scripts/deploy.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -e
+
+DOCKER_REGISTRY_HOST=ghcr.io
+
+# TODO(dmu) MEDIUM: Figure out a way to pass environment variables from github actions to
+#                   this script instead of using positional arguments
+GITHUB_USERNAME="${GITHUB_USERNAME:-$1}"
+GITHUB_PASSWORD="${GITHUB_PASSWORD:-$2}"
+CLEAR_BLOCKCHAIN="${CLEAR_BLOCKCHAIN:-$3}"
+CLEAR_BLOCKCHAIN="${CLEAR_BLOCKCHAIN:-False}"
+INITIALIZE_FROM_ALPHA="${INITIALIZE_FROM_ALPHA:-$4}"
+INITIALIZE_FROM_ALPHA="${INITIALIZE_FROM_ALPHA:-False}"
+
+docker logout $DOCKER_REGISTRY_HOST
+
+# Support github actions deploy as well as manual deploy
+if [[ -z "$GITHUB_USERNAME" || -z "$GITHUB_PASSWORD" ]]; then
+  echo "Interactive docker registry login (username=github username; password=github personal access token (not github password)"
+  docker login $DOCKER_REGISTRY_HOST
+else
+  echo "Automated docker registry login"
+  # TODO(dmu) LOW: Implement a defensive technique to avoid printing password in case of `set -x`
+  docker login --username "$GITHUB_USERNAME" --password "$GITHUB_PASSWORD" $DOCKER_REGISTRY_HOST
+fi
+
+wget https://raw.githubusercontent.com/thenewboston-developers/Node/development/docker-compose.yml -O docker-compose.yml
+
+test -f .env || touch .env
+grep -q -o MONGO_INITDB_ROOT_PASSWORD .env || echo "MONGO_INITDB_ROOT_PASSWORD=$(xxd -l 16 -p /dev/urandom)" >>.env
+grep -q -o TNB_SECRET_KEY .env || echo "TNB_SECRET_KEY=$(xxd -c 48 -l 48 -p /dev/urandom)" >>.env
+
+docker-compose pull
+
+TEMPORARY_NODE_SIGNING_KEY=0000000000000000000000000000000000000000000000000000000000000000
+TNB_NODE_SIGNING_KEY=$(docker-compose --log-level CRITICAL run --rm -e TNB_NODE_SIGNING_KEY=$TEMPORARY_NODE_SIGNING_KEY node poetry run python -m node.manage generate_signing_key)
+grep -q -o TNB_NODE_SIGNING_KEY .env || echo "TNB_NODE_SIGNING_KEY=$TNB_NODE_SIGNING_KEY" >>.env
```

Comment:
let's put prev into echo or otherwise prevent wast signing key generation if it is already in `.env` file

Reply:
@olegtropinin please, address this comment